lib/_sidebar.erb

```diff
@@ -0,0 +1,64 @@
+    <aside class="sidebar">
```

Comment:
This is the .erb that is generated from _sidebar-template.erb, I think we can remove this file from the commit.